.changeset/fluffy-cups-travel.md

```diff
@@ -0,0 +1,52 @@
+---
+'@astrojs/rss': major
+---
+
+Update RSS configuration with content collections in mind.
```

Comment:
I'd suggest putting these points last and moving the migration guide first. Coming from someone who is migrating from the previous major to this major, I'd want to know the relevant breaking changes that I need to make first. I might not know what "content collections" is.

Reply:
Understood. Redrafted and bumped down to a minor! This is now deprecated, not removed 👍